cmd/commands/start.go

```diff
@@ -125,15 +125,16 @@ func doStart(cmd *cobra.Command, args []string) {
 	}
 
 	opts := lite.StartOpts{
-		Config:        *conf,
-		PollInterval:  viper.GetInt("poll-interval"),
-		RedisURI:      viper.GetString("redis-uri"),
-		RetryInterval: viper.GetInt("retry-interval"),
-		Tick:          time.Duration(tick) * time.Millisecond,
-		URLs:          viper.GetStringSlice("sdk-url"),
-		SQLiteDir:     viper.GetString("sqlite-dir"),
-		SigningKey:    viper.GetString("signing-key"),
-		EventKey:      viper.GetStringSlice("event-key"),
+		Config:            *conf,
+		PollInterval:      viper.GetInt("poll-interval"),
+		RedisURI:          viper.GetString("redis-uri"),
+		RetryInterval:     viper.GetInt("retry-interval"),
+		Tick:              time.Duration(tick) * time.Millisecond,
+		URLs:              viper.GetStringSlice("sdk-url"),
+		SQLiteDir:         viper.GetString("sqlite-dir"),
+		SigningKey:        viper.GetString("signing-key"),
+		EventKey:          viper.GetStringSlice("event-key"),
+		EnablePersistence: !config.IsEnvVarFalsy("INNGEST_EXPERIMENTAL_ENABLE_PERSISTENCE"),
```

Comment:
It kind of feels like a weird pattern to use an falsy value a truthy sounding env var that is already truthy by default. It makes the logic more complex. Like "Persistence is enabled by default. Do you want to disable persistence? Set the enable persistence var to false."

Since this is just on `inngest start` which enables persistence by default, I think we should just call this `INNGEST_EXPERIMENTAL_DISABLE_PERSISTENCE` - the logic becomes simpler.

Reply:
👍 Agree - switching this to `INNGEST_EXPERIMENTAL_DISABLE_PERSISTENCE` feels good.

cc @amh4r - just this and LGTM